timelock-impl/src/main/java/com/palantir/atlasdb/timelock/paxos/PaxosTimestampBoundStore.java

```diff
@@ -57,23 +56,20 @@
     private final PaxosProposer proposer;
     private final PaxosLearner knowledge;
 
-    private final List<PaxosAcceptor> acceptors;
-    private final List<PaxosLearner> learners;
+    private final Map<PaxosAcceptor, ExecutorService> acceptors;
```

Comment:
as an aside, in a later PR, I think we need a better way of handling this sort of stuff without having to worry about wiring everywhere. e.g. some class like:

```java
interface PaxosRemotes<SERVICE> {
  SERVICE local();
  Set<SERVICE> remotes();

  // this can just delegate to `PaxosQuorumChecker`
  <RESULT extends PaxosResponse> PaxosResponses<RESULT> compute(
      ExecutorService executorService, 
      Function<SERVICE, PaxosResponses<RESULT> function);

  // OR
  ExecutorService executorService();
  <RESULT extends PaxosResponse> PaxosResponses<RESULT> compute(
      Function<SERVICE, PaxosResponses<RESULT> function);
}
```

benefits:
* can easily add in code that schedules calls to locals on the calling thread for example
* can experiment with adding "host" limits e.g. a better multiplexing completion service
etc.

at that point, callers largely don't have to worry about any such changes since it's pretty transparent. For a later PR though.